### PR TITLE
Separate etherlime into two packages; add lerna configs; configure tests

### PR DESCRIPTION
**Pull request by desimira**

### CODE REVIEW

File: packages/etherlime/cli.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 const commands = require('./cli-commands/commands');
 const { exec } = require('child_process');
-const globalExceptionHandling = require('./utils/global-exception-handling');
+const globalExceptionHandling = require('etherlime-utils');
```
Comment (Perseverance) [inline packages/etherlime/cli.js]: This does not seem correct to me

File: packages/etherlime/deployed-contract/deployed-contract-wrapper.js [hunk 2]
```diff
@@ -1,8 +1,5 @@
 const ethers = require('ethers');
-const isAddress = require('./../utils/address-utils').isAddress;
-const isSigner = require('./../utils/signer-utils').isSigner;
-const isValidContract = require('./../utils/contract-utils').isValidContract;
-const colors = require('./../utils/colors');
+const {isAddress, isSigner, isValidContract, colors} = require('etherlime-utils')
```
Comment (Perseverance) [inline packages/etherlime/deployed-contract/deployed-contract-wrapper.js]: Half the places have spaces around the curly brackets and half of them do not, please stick with spaces around the curly brackets

File: test/etherlime-utils/utils/linking-utils.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ describe('Linking library tests', () => {
         Object.prototype.LinkedList = "0x2Be52D5d7A73FC183cF40053B95beD572519EBbC";
 
         let bytecode = linkLibrary(libraries, Greetings.bytecode);
-
+        delete Object.prototype.LinkedList
```
Comment (ochikov) [inline test/etherlime-utils/utils/linking-utils.js]: Should we delete this after the assert ?

File: test/etherlime/cli-commands/commands/commands.js [hunk 4]
```diff
@@ -5,21 +5,22 @@ chai.use(chaiAsPromised);
...
 
 
 describe('root calling cli commands', () => {
 
-    it('should throw err if init cli command failed', async function () {
-        let expectedOutput = "Command failed:"
-        let childProcess = await runCmdHandler(`etherlime init`, expectedOutput);
-        assert.include(JSON.stringify(childProcess), expectedOutput)
-    });
+    // it('should throw err if init cli command failed', async function () {
```
Comment (ochikov) [inline test/etherlime/cli-commands/commands/commands.js]: Should we uncomment this, or it is broken and we are going to fix it later ?

**Pull request merged**